TMDbLib/Objects/TvShows/TvShow.cs

```diff
@@ -71,9 +71,15 @@ public class TvShow
         [JsonProperty("last_air_date")]
         public DateTime? LastAirDate { get; set; }
 
+        [JsonProperty("last_episode_to_air")]
+        public TvEpisode LastEpisodeToAir { get; set; }
```

Comment:
TvEpisode also has an `account_states` which the Api docs don't describe.. Do we have a more fitting type, or should we split TvEpisode into one that's account-enabled and one that's not?

Reply:
I found that i was almost the same type used for TvGroupEpisodes and split it up, creating a base class.